packages/realm-react/example/package.json

```diff
@@ -3,33 +3,32 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "android": "react-native run-android",
-    "ios": "react-native run-ios",
+    "android": "concurrently -n metro,rn npm:start 'react-native run-android'",
```

Comment:
What is this doing?

Reply:
The React Native CLI is not happy to be run when it's not a located in the `node_modules` of the app.
More specifically the metro server which pops up has an issue with this, which is why I suggest migrating to something that doesn't "pop up" the Metro terminal.